### PR TITLE
Add support for new dialog files in packager

### PR DESCRIPTION
**Pull request by pvanderknyff**
Adds XSD files for modular connection dialogs as well. Will add some unit tests as well.

Opening this against dev-2020.2 since this won't be able to be used until that release.

### CODE REVIEW

File: connector-packager/connector_packager/xml_parser.py [hunk 1]
```diff
@@ -153,6 +153,19 @@ def parse_file(self, file_to_parse: ConnectorFile) -> bool:
                 # add new file to list
                 self.file_list.append(new_file)
 
+                # If connection-metadata, make sure that connection-fields file exists
+                if child.tag == "connection-metadata":
```
Comment (rosswbrown) [inline connector-packager/connector_packager/xml_parser.py]: We're inconsistent about single vs double quote, seems like mostly single, match the file I suppose.

File: connector-packager/connector_packager/xml_parser.py [hunk 2]
```diff
@@ -153,6 +153,19 @@ def parse_file(self, file_to_parse: ConnectorFile) -> bool:
...
+                    connection_metadata_exists = False
+
+                    for xml_file in self.file_list:
+                        if xml_file.file_type == "connection-fields":
+                            connection_metadata_exists = True
```
Comment (rosswbrown) [inline connector-packager/connector_packager/xml_parser.py]: connection_fields_exists

File: connector-packager/connector_packager/xml_parser.py (hunk 1 above)
Comment (pvanderknyff) [reply connector-packager/connector_packager/xml_parser.py]: Changed to single quote

File: connector-packager/connector_packager/xml_parser.py (hunk 2 above)
Comment (pvanderknyff) [reply connector-packager/connector_packager/xml_parser.py]: Changed

File: connector-packager/connector_packager/version.py [hunk 3]
```diff
@@ -1,2 +1,2 @@
 __version__ = '1.0.0'
-__min_version_tableau__ = '2019.4'
+__default_min_version_tableau__ = '2019.4'
```
[the remark this answers is not in the document]
Comment (pvanderknyff) [reply connector-packager/connector_packager/version.py]: With my changes, this is the default. Most connectors at the moment can work with 2019.4. I added a method that determines the minimum version, and if there are files that use the new modular connection dialog framework we stamp 2020.2 instead.

**Pull request merged**